index.js

```diff
@@ -1008,31 +1053,42 @@ Multi.prototype.send_command = function (command, args, index, cb) {
     });
 };
 
-Multi.prototype.exec = Multi.prototype.EXEC = function (callback) {
+Multi.prototype.exec_transaction = function (callback) {
     var self = this;
+    var len = this.queue.length;
+    var cb;
```

Comment:
Sorry I think we've got wires crossed here! 

Yes, running the empty multi/exec works without any problems and responds with `null, []` (correctly), but if we're talking about speed improvements then preventing the unnecessary round-trip to redis will make things faster! 

If this isn't in the node_redis code, then users (like me) will have to continue checking that `multi.queue.length > 1` all over their application code. It's a simple and easy optimisation that can be put into node_redis, so hopefully it can be added.

Slightly related to this (multi queue lengths!), did you have any thoughts about adding `atomicBatch`? So that'd be extracting the single command from a multi if there's just one in the queue (if `multi.queue.length === 2`)